bin/wait_then_run

```diff
@@ -30,7 +30,5 @@ export PYTHONPATH=$(pwd)
 if [ ! -z $* ]; then
   $*
 else
-  python3 hmt_escrow/job.py
-  python3 hmt_escrow/storage.py
-  python3 hmt_escrow/eth_bridge.py
+  python3 -m unittest discover -s test/hmt_escrow
```

Comment:
I would've created two PRs, one with the refactoring, and one with the two-liner for the "Adding support to decide whether final results must be encrypted".

Reply:
@igervasiychuk , I will close this PR in favor of PR split to other 3:
- https://github.com/humanprotocol/hmt-escrow/pull/328
- https://github.com/humanprotocol/hmt-escrow/pull/329